src/applications/gi/containers/SearchPage.jsx

```diff
@@ -67,7 +67,9 @@ export class SearchPage extends React.Component {
       'yellowRibbonScholarship',
       'principlesOfExcellence',
       'eightKeysToVeteranSuccess',
-      'stemOffered'
+      'stemOffered',
+      'priorityEnrollment',
```

Comment:
These new query params need to be in this array to build the correct XHR.

Reply:
Should we use constants and proxies here?